src/core/Stage.tsx

```diff
@@ -54,6 +55,7 @@ export function Stage({
   shadowBias = 0,
   contactShadowBlur = 2,
```

Comment:
do you think we should move the `contactShadowX` props to an object instead of individual props? I'm concerned the prop list will get out of hand... then you could provide any of the valid `ContactShadow` props...